src/app/Http/Controllers/BusinessTripController.php

```diff
@@ -77,4 +85,203 @@ public function confirm() {
+            return response()->json(['error' => 'PDF template file not found'], 404);
+        }
+
+        $outputPath = Storage::disk('pdf-exports')
+            ->path(uniqid('', true) . '.pdf');
+
+        $data = [];
+        switch ($documentType) {
+            case 'cestne_vyhlasenie_k_zahranicnej_pc.pdf':
+                $data = [
+                    'order_number' => $trip->sofia_id,
+                    'trip_duration' => '?', //$trip->'?', // tu potrebujem pomôcť, ktorý stĺpec z databázy to je
+                    'adress' => $trip->place,
+                    'name' => $trip->user->first_name . ' ' . $trip->user->last_name,
+                ];
+                break;
+
+            case DocumentType::COMPENSATION_AGREEMENT:
+                $data = [
+                    'first_name' => $trip->user->first_name,
+                    'last_name' => $trip->user->last_name,
+                    'academic_degree' => $trip->user->academic_degrees,
+                    'address' => $trip->user->address,
+                    'contribution1' => '?', // v databáze neexistuje prepojenie na tabuľku s prínosmi pre fakultu
+                    'contribution2' => '?', // -||-
+                    'contribution3' => '?', // -||-
+                    'department' => $trip->user->department,
+                    'place' => $trip->country->name . ', ' . $trip->place,
+                    'datetime_start' => $trip->datetime_start->format('Y-m-d'),
+                    'datetime_end' => $trip->datetime_end->format('Y-m-d'),
+                    'transport' => $trip->transport->name,
+                    'trip_purpose' => $trip->tripPurpose->name,
+                    'fund' => $trip->sppSymbol->fund,
+                    'functional_region' => $trip->sppSymbol->functional_region,
+                    'financial_centre' => $trip->sppSymbol->financial_centre,
+                    'spp_symbol' => $trip->sppSymbol->spp_symbol,
+                    'account' => $trip->sppSymbol->account,
+                    'grantee' => $trip->sppSymbol->grantee,
+                    'iban' => $trip->iban,
+                    'bank_name' => '?',
+                    'incumbent_name2' => '?', // sem sa bude dopĺnať meno a priezvisko z tabuľky staff
+                    'incumbent_name1' => '?', // -||-
+                    'position_name1' => '?',  // -||-
+                    'position_name2' => '?',  // -||-
+                    'contribution1_text' => '?', // v databáze neexistuje prepojenie na tabuľku s prínosmi pre fakultu
+                    'contribution2_text' => '?', // -||-
+                    'contribution3_text' => '?', // -||-
+                ];
+                break;
+
+            case DocumentType::CONTROL_SHEET:
+                $data = [
+                    'spp_symbol' => $trip->sppSymbol->spp_symbol,
+                    'expense_estimation' => $trip->expense_estimation,
```

Comment:
tu tiez $trip->conference_fee->amount, je to iba nespravne nazvane v pdf template, podla mna